Doc/tutorial/introduction.rst

```diff
@@ -189,6 +189,16 @@ the first quote::
    >>> print(r'C:\some\name')  # note the r before the quote
    C:\some\name
 
+Note that escaping quotes in raw strings will keep the backslash::
```

Comment:
Maybe to be explicit, it could be something like

```suggestion
Note that, unlike the non–raw string case, escaping quotes in raw strings will keep the backslash::
```

Reply:
Thanks for making the change! This looks good, but let's move it to the FAQ section?

My reasoning is based on the description of raw strings it's not really surprising that the backslash is preserved. It's only really surprising in the context of the end quote situation, where the backslash "escapes" the quote for the tokeniser, but not for the actual value.

That is, if you read:
> A raw string ending with an odd number of backslashes will escape the string's quote

you may come away with the impression that raw strings are raw except for quotes or something. I guess a concrete phrasing would be adding the following to the end of the FAQ entry:

```
Note that while a backslash will "escape" a quote for the purposes
of determining where the raw string ends, the backslash remains
present in the value of the raw string::

>>> r'backslash\'preserved'
"backslash\\'preserved"
```

Reply:
Okay, there's prior art. Here's the relevant words from [Lexical Analysis](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals):

> Even in a raw literal, quotes can be escaped with a backslash, but the backslash remains in the result; for example, r"\"" is a valid string literal consisting of two characters: a backslash and a double quote; r"\" is not a valid string literal (even a raw string cannot end in an odd number of backslashes). Specifically, a raw literal cannot end in a single backslash (since the backslash would escape the following quote character). Note also that a single backslash followed by a newline is interpreted as those two characters as part of the literal, not as a line continuation.

Reply:
Hmm, I'm not too certain what the tokenization / determining where the raw string ends means too well, but I've commited the changes (and added a link to the reference) for the time being.